app/Http/routes.php

```diff
@@ -10,6 +10,9 @@
 Route::get('/developer', 'ApiPagesController@index');
 Route::get('/developer/myapp', 'ApiPagesController@myApp');
 Route::get('/developer/myapp/new', 'ApiPagesController@createNewApp');
+Route::post('/developer/myapp/new/', function() {
+    echo "App created successfully";
+});
```

Comment:
**Code Functionality - Critcal**

Is there any reason we have it echoing a sentence on a post request?